### ISSUE TITLE
Add Version option

### ISSUE THREAD

**Issue by cas--** [nvbn/everpad#115]
It would be handy to have version information at least as a console option, if not also an About entry in the indicator. 

Mainly its useful to know what version I have installed compared to latest releases in git and PPA plus when reporting bugs have that extra information to hand.

**Comment by hdave** [comment]
A workaround when running from the PPA is:

`dpkg -s everpad`

**Comment by fmoo** [comment]
Yeah, might be nice to have an about page with something people can copy-paste into the issue tracker that lists 
- the version for each running service (lens, providier, indicator)
- the versions of all the running dependencies (e.g., from **version** in the modules themselves)

Also might be great to export this information over dbus.  Not sure if there's a standard for this or no.

**Comment by e2jk** [comment]
```
$ everpad -v
Everpad version: 2.5
API version: 6
Schema version: 5
```

Also works with `everpad --version`.
Please close this issue, will allow the developers to focus on open issues.
